apps/dot_ext/tests/test_views.py

```diff
@@ -12,6 +14,7 @@
 
 
 class TestApplicationUpdateView(BaseApiTest):
+    @skip("Skipping until restricted scopes are reenabled. They were disabled for https://issues.hhsdevcloud.us/browse/CBBP-881")
```

Comment:
Can you just delete this test instead of skipping it? We can write a test that matches the implementation if we re-implement this.